configure

```diff
@@ -12735,7 +12735,7 @@ $as_echo "no" >&6; }
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 
-# check for openpty and forkpty
+# check for openpty, login_tty, and forkpty
```

Comment:
I have edited `configure.ac` and generated `configure` using `autoconf` now. Are there any steps that I might have missed/done improperly? When I generated the `configure` script, it added the `runstatedir` option to it which was not there before; please let me know if that was not supposed to happen/how to fix it. Also, it says that this branch has "conflicts that must be resolved"; Modules/clinic/posixmodule.c.h is the conflicting file; what is the best way to fix this problem; I want to avoid the XY problem this time!